src/nft/utils/scrollToTop.ts

```diff
@@ -1,3 +1,5 @@
-const DESKTOP_OFFSET = 420
-
-export const scrollToTop = () => window.scrollTo({ top: DESKTOP_OFFSET })
+export const scrollToTop = () => {
+  window.document.getElementById('nft-anchor')?.scrollIntoView({
```

Comment:
https://github.com/Uniswap/interface/blob/main/src/pages/App.tsx#L139

Reply:
this needs to scroll to a specific point in the page, so that's why I added the anchor